pyogrio/tests/test_geopandas_io.py

```diff
@@ -297,7 +297,10 @@ def test_read_datetime(datetime_file, use_arrow):
 
 @pytest.mark.filterwarnings("ignore: Non-conformant content for record 1 in column ")
 @pytest.mark.requires_arrow_write_api
-def test_read_datetime_tz(datetime_tz_file, tmp_path, use_arrow):
+def test_write_datetime_tz(datetime_tz_file, tmp_path, use_arrow):
+    if use_arrow and __gdal_version__ < (3, 11, 0):
+        pytest.skip("Arrow datetime handling improved in GDAL >= 3.11")
```

Comment:
Can we still test what behaviour you get with the current GDAL?

Reply:
I replaced all skip's by a test on the result of GDAL <3.11